test/quicer_SUITE.erl

```diff
@@ -1344,6 +1379,43 @@ ping_pong_server_stm_loop(L, Conn, Stm) ->
       quicer:close_listener(L)
   end.
 
+ping_pong_server_dgram(Owner, Config, Port) ->
+  Opts = default_listen_opts(Config) ++ [{datagram_receive_enabled, 1}],
+  case quicer:listen(Port, Opts) of
+    {ok, L} ->
+      Owner ! listener_ready,
+      {ok, Conn} = quicer:accept(L, [], 5000),
+      {ok, Conn} = quicer:handshake(Conn),
+      {ok, Stm} = quicer:accept_stream(Conn, []),
+      ping_pong_server_dgram_loop(L, Conn, Stm);
+    {error, listener_start_error, R} ->
+      ct:pal("Failed to start listener:~p , retry ...", [R]),
+      timer:sleep(100),
+      ping_pong_server_dgram(Owner, Config, Port)
+  end.
+
+ping_pong_server_dgram_loop(L, Conn, Stm) ->
+  receive
+    {quic, <<"ping">>, _, _, _, _} ->
+      ct:pal("send pong"),
+      {ok, 4} = quicer:send(Stm, <<"pong">>),
+      ping_pong_server_stm_loop(L, Conn, Stm);
```

Comment:
well, this should be `ping_pong_server_dgram_loop`. I think this is the reason makes the test unstable in github CI.

Reply:
:man_facepalming: 

Thanks a lot. I have fixed it